docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -45,6 +45,35 @@ Add `-javaagent` argument to command line in which you start your app. eg:
  ```shell
  java -javaagent:/path/to/skywalking-agent/skywalking-agent.jar -jar yourApp.jar
  ```
+ 
+## Table of Agent Configuration Properties
+
+property key | Description | Default |
+----------- | ---------- | --------- | 
+`agent.namespace` | Namespace isolates headers in cross process propagation. The HEADER name will be `HeaderName:Namespace`. | Not set | 
+`agent.service_name` | Application(5.x)/Service(6.x) code is showed in sky-walking-ui. Suggestion: set an unique name for each application, one application's nodes share the same code. | `Your_ApplicationName` |
```

Comment:
`sky-walking` -> `skywalking`

Reply:
`Suggestion: set an unique name for each application, one application's nodes share the same code.` -> `Suggestion: set a unique name for each service, service instance nodes share the same code`